sklearn/notebooks/hp_tuning_auto_mpg/ml_engine_hp_tuning_training.ipynb

```diff
@@ -0,0 +1,804 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#!/usr/bin/env python\n",
```

Comment:
seems a little off to start a notebook like this. i understand we put a license at the top of our code, maybe here it can be in a text block rather than a code block?